ios/brave-ios/Sources/AIChat/Subscription/SDK/AppStoreSDK.swift

```diff
@@ -45,10 +46,12 @@ public class AppStoreReceipt {
       fetcher.refreshReceipt { error in
         DispatchQueue.main.async {
           if let error = error {
+            Logger.module.error("[AppStoreReceipt] - Error Refreshing Receipt: \(error)")
             continuation.resume(throwing: error)
             return
           }
 
+          Logger.module.debug("[AppStoreReceipt] - Receipt Refreshed Successfully")
```

Comment:
debug logs dont show up in Console.app in release builds, not sure if they'll show up in OSLogStore.  Best to test in a local Release build